Sources/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -280,6 +280,23 @@ public extension Array {
             value += slice
         }
     }
+    
+    
+    /// SwifterSwift: Returns an array of slices of lenght "size" from the array.  If array can't be split evenly, the final slice will be the remaining elements.
+    ///
+    /// - Parameters:
+    ///   - size: The size of the slices to be returned.
+    public func split(size: Int) -> [ArraySlice<Element>]? {
```

Comment:
I'm a little concerned about the memory implications of returning an ArraySlice here. I think returning an Array would be a safer API for the user.